### ISSUE TITLE
add “show correct answer” button to the “Instructor Info” box

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#1560]
The current “Instructor Info” box can show the correct answer as raw JSON, but for testing purposes it would be good to be able to show the rendered correct answer.

One way to do this would be to always render the correct answer box for instructors, but set it to `visible=“false”` and then change this attribute via a button in the Instructor Info box. Retaining the ability to see the raw JSON is probably also helpful for debugging. Perhaps it would even be good to add the ability to display all the raw JSON info (`params`, `submitted_answer`, `correct_answer`, etc)?

**Comment by coatless** [comment]
👍 

I think this can close out #1114

**Comment by mwest1066** [comment]
@coatless good point, this is essentially a dup of #1114

**Comment by mwest1066** [comment]
See also #507

**Comment by trombonekenny** [comment]
Similar to #800?

**Comment by mwest1066** [comment]
Yep, same as #800. With the addition of being able to also show `params`, `submitted_answer`, `correct_answer`, etc.
